sdks/java/container/boot.go

```diff
@@ -103,7 +103,17 @@ func main() {
 		filepath.Join(jarsDir, "slf4j-jdk14.jar"),
 		filepath.Join(jarsDir, "beam-sdks-java-harness.jar"),
 	}
+
+  var has_worker_experiment = strings.Contains(options, "use_staged_dataflow_worker_jar")
```

Comment:
nit: go uses camelCase for local variables. You also need to run "gofmt -w ." to fix the indentation.

Reply:
Fixed, thanks~